packages/react/src/auth/useSession.ts

```diff
@@ -0,0 +1,37 @@
+import { useApi } from "../../src/GadgetProvider";
+import { useGet } from "../../src/useGet";
+
+export interface GadgetSession {
```

Comment:
I had to type this in `@gadgetinc/auth` too. Maybe this needs to be moved to `api-client-core`